changelog.d/5151.misc

```diff
@@ -0,0 +1 @@
+Improve threads rendering in the main timeline
```

Comment:
should we mention what the improvement was in the changelog? (stopping threads from breaking message chains)